src/NameRegistry.sol

```diff
@@ -223,44 +243,29 @@ contract NameRegistry is
      */
     mapping(bytes32 => uint256) public timestampOf;
 
-    /**
-     * @notice Maps each uint256 representation of an fname to the time at which it expires
-     * @dev    Occupies slot 4
-     */
-    mapping(uint256 => uint256) public expiryOf;
-
     /**
      * @notice The address that funds can be withdrawn to
-     * @dev    Occupies slot 5
+     * @dev    Occupies slot 4
      */
     address public vault;
 
     /**
      * @notice The address that names can be reclaimed to
-     * @dev    Occupies slot 6
+     * @dev    Occupies slot 5
      */
     address public pool;
 
     /**
-     * @notice Maps each uint256 representation of an fname to the address that can recover it
-     * @dev    Occupies slot 7
-     */
-    mapping(uint256 => address) public recoveryOf;
-
-    /**
-     * @notice Maps each uint256 representation of an fname to the timestamp of the recovery
-     *         attempt or zero if there is no active recovery.
-     * @dev    Occupies slot 8
+     * @notice Maps each uint256 representation of an fname to registration metadata
+     * @dev    Occupies slot 6
      */
-    mapping(uint256 => uint256) public recoveryClockOf;
+    mapping(uint256 => RegistrationMetadata) public registrationInfo;
```

Comment:
can you update this to `registrationMetadataOf` to stick with the existing conventions for map names?